napari/_vispy/_tests/test_utils.py

```diff
@@ -4,16 +4,24 @@
 from napari._vispy.quaternion import quaternion2euler
 
 
-def test_quaternion2euler(qtbot):
+def test_quaternion2euler():
```

Comment:
you can use `pytest.mark.parametrize` here (see example https://github.com/napari/napari/blob/102a7e8f845893c874d2b86f9371d41130100b89/napari/utils/transforms/_tests/test_transforms.py#L8-L9) to pass the list of `angles` and then you don't need a for loop inside the test :-)

Reply:
Nice trick ! :) I added it to the test !